vm/systemSmartContracts/staking.go

```diff
@@ -401,6 +430,13 @@ func (r *stakingSC) stake(args *vmcommon.ContractCallInput, onlyRegister bool) v
 		return vmcommon.UserError
 	}
 
+	err = r.eei.UseGas(r.gasCost.MetaChainSystemSCsCost.Stake +
```

Comment:
this computation for stake is different than on the auctionSC. Should it be different?

Reply:
yes. as you make a store in staking as well. not only in auction.

Reply:
ok